.github/workflows/README.md

```diff
@@ -12,6 +13,71 @@ The following workflows can be found here:
 - [Java Gradle Plugin](https://github.com/bakdata/ci-templates/tree/main/.github/workflows#java-gradle-plugin)
 - [Java Gradle Release](https://github.com/bakdata/ci-templates/tree/main/.github/workflows#java-gradle-release)
 
+## Docker Build and Publish
+
+This workflow will use a Dockerfile to build and push images to any container registry.
+
+### Prerequisites
+
+This workflow requires a Dockerfile located in the repository.
+
+### Dependencies
+
+This workflow is built from multiple composite actions listed below:
+
+- [docker-build](https://github.com/bakdata/ci-templates/tree/main/actions/docker-build)
+- [docker-publish](https://github.com/bakdata/ci-templates/tree/main/actions/docker-publish)
+
+### Input Parameters
+
+| Name                | Required |            Default Value            |  Type  | Description                                                                                                                                         |
+| ------------------- | :------: | :---------------------------------: | :----: | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| docker-context      |    ❌    |                 "."                 | string | The docker context.                                                                                                                                 |
+| dockerfile-path     |    ❌    |            "Dockerfile"             | string | Path to the Dockerfile.                                                                                                                             |
+| docker-registry     |    ❌    |                 ""                  | string | Host where the image should be pushed to.                                                                                                           |
+| image-artifact-name |    ❌    |          "image-artifact"           | string | Name of the artifact that contains the Docker image.tar file to push, see https://github.com/actions/upload-artifact (Default is 'image-artifact'). |
+| image-name          |    ❌    | ${{ github.event.repository.name }} | string | Name of Docker image (Default is the repository name).                                                                                              |
```

Comment:
Please write the default without variable braces, just `github.event.repository.name`